pkg/apis/pxc/v1/pxc_backup_types.go

```diff
@@ -53,7 +56,7 @@ type PXCBackupStatus struct {
 	State                 PXCBackupState          `json:"state,omitempty"`
 	CompletedAt           *metav1.Time            `json:"completed,omitempty"`
 	LastScheduled         *metav1.Time            `json:"lastscheduled,omitempty"`
-	Destination           string                  `json:"destination,omitempty"`
+	Destination           PXCDestination          `json:"destination,omitempty"`
```

Comment:
Maybe to name this `BackupDestination` or `PXCBackupDestination`, only `PXCDestination` is not that clear that it relates to a backup.

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1600/commits/03c867973553d3ff0f3da368fd1ba864f9c38748